docs/devel/testing.md

```diff
@@ -98,18 +98,24 @@ You can set [go flags](https://golang.org/cmd/go/) by setting the
 
 ### Run unit tests from certain packages
 
-The `hack/test-go.sh` script accepts packages as arguments; the
-`k8s.io/kubernetes` prefix is added automatically to these:
+`make test` accepts packages as arguments; the `k8s.io/kubernetes` prefix is
+added automatically to these:
 
 ```sh
-hack/test-go.sh pkg/api             # run tests for pkg/api
-hack/test-go.sh pkg/api pkg/kubelet # run tests for pkg/api and pkg/kubelet
+make test WHAT=pkg/api              # run tests for pkg/api
+make test WHAT=pkg/api pkg/kubelet  # run tests for pkg/api and pkg/kubelet
```

Comment:
Based on the comment below, this one is incorrect (because it doesn't have quotes). If so, please remove it, otherwise please change the comment below